utils/test-support/src/main/java/org/apache/brooklyn/test/support/FlakyRetryAnalyser.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.brooklyn.test.support;
+
+import org.testng.IRetryAnalyzer;
+import org.testng.ITestResult;
+
+public class FlakyRetryAnalyser implements IRetryAnalyzer {
+    private int retryCount = 0;
+    private static final int maxRetryCount = 3;
```

Comment:
Constants, `private static final`, are usually all caps: `MAX_RETRY_COUNT`.
Within the context of a 'retryAnalyzer' you could get away with `count` and `MAX`. They are both private.

Reply:
Good point @kemitix, code is updated